### ISSUE TITLE
Label possition conflict when using doughnutlabel plugin

### ISSUE THREAD

**Issue by Seb33300** [emn178/chartjs-plugin-labels#67]
When using your plugin in association with [doughnutlabel plugin](https://github.com/ciprianciurea/chartjs-plugin-doughnutlabel), arc label and % sign are not properly positioned:

Without doughnutlabel: OK
![image](https://user-images.githubusercontent.com/915273/44085052-9f20d7c4-9fe2-11e8-96c7-6baa2b9f2cbd.png)

With doughnutlabel: KO
![image](https://user-images.githubusercontent.com/915273/44085093-b9636606-9fe2-11e8-8a0e-64d8f77507a6.png)

``` html
<canvas id="statistics-chart" height="280"></canvas>

<script>

    var statisticsQuantities = [
        2,
        1,
        2
    ];

    new Chart(document.getElementById('statistics-chart').getContext('2d'), {
        type: 'doughnut',
        data: {
            labels: ['Publications des administrateurs', 'Publications des élèves', 'Trombinoscopes'],
            datasets: [{
                data: [
                    0.1,
                    0.05,
                    0.13,
                    0.72
                ],
                backgroundColor: ['rgba(107, 201, 199, 1)', 'rgba(39, 181, 179, 1)', 'rgba(255, 210, 3, 1)', 'rgba(0, 0, 0, 0)'],
                hoverBorderColor: ['rgba(0, 0, 0, 0)', 'rgba(0, 0, 0, 0)', 'rgba(0, 0, 0, 0)', 'rgba(0, 0, 0, 0)']
            }]
        },
        options: {
            legend: { position: 'bottom' },
            tooltips: {
                displayColors: false,
                callbacks: {
                    title: function (tooltipItems, data) {
                        var title = '';
                        tooltipItems.forEach(function (tooltipItem) {
                            title = data.labels[tooltipItem.index] ? data.labels[tooltipItem.index] : 'Reste à faire';
                        });
                        return title;
                    },
                    label: function (tooltipItem, data) {
                        return statisticsQuantities[tooltipItem.index] ? 'Quantité : ' + statisticsQuantities[tooltipItem.index] : '';
                    },
                    afterLabel: function (tooltipItem, data) {
                        return 'Taux : ' + (data.datasets[tooltipItem.datasetIndex].data[tooltipItem.index] * 100) + '%';
                    }
                }
            },
            rotation: 0.1 * Math.PI,
            pieceLabel: [
                {
                    render: function (args) { return statisticsQuantities[args.index] ? statisticsQuantities[args.index].toString() : ''; },
                    fontSize: 18,
                    fontStyle: 'bold',
                    fontColor: '#fff'
                },
                {
                    render: 'percentage',
                    fontSize: 18,
                    fontStyle: 'bold',
                    fontColor: ['rgba(107, 201, 199, 1)', 'rgba(39, 181, 179, 1)', 'rgba(255, 210, 3, 1)', 'rgba(0, 0, 0, 0)'],
                    arc: true,
                    position: 'outside'
                }
            ],
            plugins: {
                doughnutlabel: {
                    labels: [
                        {
                            text: 'taux avancement',
                            font: {
                                size: 15,
                                weight: 'bold'
                            }
                        },
                        {
                            text: 'projet',
                            font: {
                                size: 15,
                                weight: 'bold'
                            }
                        },
                        {
                            text: '28%',
                            font: {
                                size: 40,
                                weight: 'bold'
                            }
                        }
                    ]
                }
            }
        }
    });

</script>
```

**Comment by emn178** [comment]
this should be fixed in v0.15.0. please try again.

**Comment by Seb33300** [comment]
Similar bug reappeared with 1.0:

![image](https://user-images.githubusercontent.com/915273/44620660-92834b80-a8c2-11e8-91f4-6d5d332e2fbd.png)

``` html
<script>

    var statisticsQuantities = [
        1,
        0,
        7
    ];

    new Chart(document.getElementById('statistics-chart').getContext('2d'), {
        type: 'doughnut',
        data: {
            labels: ['Publications des administrateurs', 'Publications des élèves', 'Trombinoscopes'],
            datasets: [{
                data: [
                    0.05,
                    0,
                    0.64,
                    0.32
                ],
                backgroundColor: ['rgba(107, 201, 199, 1)', 'rgba(39, 181, 179, 1)', 'rgba(255, 210, 3, 1)', 'rgba(0, 0, 0, 0)'],
                hoverBorderColor: ['rgba(0, 0, 0, 0)', 'rgba(0, 0, 0, 0)', 'rgba(0, 0, 0, 0)', 'rgba(0, 0, 0, 0)']
            }]
        },
        options: {
            legend: { position: 'bottom' },
            tooltips: {
                displayColors: false,
                callbacks: {
                    title: function (tooltipItems, data) {
                        var title = '';
                        tooltipItems.forEach(function (tooltipItem) {
                            title = data.labels[tooltipItem.index] ? data.labels[tooltipItem.index] : 'Reste à faire';
                        });
                        return title;
                    },
                    label: function (tooltipItem, data) {
                        return statisticsQuantities[tooltipItem.index] ? 'Quantité : ' + statisticsQuantities[tooltipItem.index] : '';
                    },
                    afterLabel: function (tooltipItem, data) {
                        return 'Taux : ' + (data.datasets[tooltipItem.datasetIndex].data[tooltipItem.index] * 100) + '%';
                    }
                }
            },
            rotation: 0.1 * Math.PI,
            plugins: {
                labels: [
                    {
                        render: function (args) { return statisticsQuantities[args.index] ? statisticsQuantities[args.index] : ''; },
                        fontSize: 18,
                        fontStyle: 'bold',
                        fontColor: '#fff'
                    },
                    {
                        render: 'percentage',
                        fontSize: 18,
                        fontStyle: 'bold',
                        fontColor: ['rgba(107, 201, 199, 1)', 'rgba(39, 181, 179, 1)', 'rgba(255, 210, 3, 1)', 'rgba(0, 0, 0, 0)'],
                        arc: true,
                        position: 'outside'
                    }
                ],
                doughnutlabel: {
                    labels: [
                        {
                            text: 'taux avancement',
                            font: {
                                size: 15,
                                weight: 'bold'
                            }
                        },
                        {
                            text: 'projet',
                            font: {
                                size: 15,
                                weight: 'bold'
                            }
                        },
                        {
                            text: '68%',
                            font: {
                                size: 40,
                                weight: 'bold'
                            }
                        }
                    ]
                }
            }
        }
    });

</script>

```

**Comment by emn178** [comment]
this should be fixed in v1.0.1
